src/Tracer.php

```diff
@@ -213,9 +230,9 @@ private function taskId(TestMethod $testMethod): int
             return 0;
         }
 
-        $matches=[];
-        $matchCount = \preg_match('/@task_id\s+(\d+)/', $docComment, $matches);
+        $matches = [];
+        $matchCount = preg_match('/@task_id\s+(\d+)/', $docComment, $matches);
 
-        return $matchCount >= 1 ? (int)$matches[1] : 0;
+        return $matchCount >= 1 ? (int) $matches[1] : 0;
```

Comment:
Please describe, so I understand this habit: Why does separating the operand from the type cast help with clarifying the expression? In my eyes it unclarifies that the type conversion only happens to `$matches[1]` and nothing else. With the space this becomes more of a thinking experiment than an immediate pattern recognition thing.

Reply:
This is from the [Doctrine coding standard](https://www.doctrine-project.org/projects/doctrine-coding-standard/en/13.0/reference/index.html):
> Add spaces after a type cast $foo = (int) '12345';

I do not really care on my side, but I know that some persons find it more readable as it avoids a pile of chars.